frontend/__tests__/components/catalog.spec.tsx

```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import * as _ from 'lodash-es';
 import { mount, ReactWrapper } from 'enzyme';
 
-import { CatalogTile } from '@patternfly/react-catalog-view-extension/dist/js/components/CatalogTile';
-import { VerticalTabsTab } from '@patternfly/react-catalog-view-extension/dist/js/components/VerticalTabs';
-import { FilterSidePanelCategoryItem } from '@patternfly/react-catalog-view-extension/dist/js/components/FilterSidePanel/FilterSidePanelCategoryItem';
+import { CatalogTile } from '@patternfly/react-catalog-view-extension';
```

Comment:
Ok, then lets do this:
`import { 
CatalogTile, 
VerticalTabsTab, 
FilterSidePanelCategoryItem 
} from '@patternfly/react-catalog-view-extension';`

Reply:
All set.